### DISCUSSION TITLE
手机系统更新后，Clash Meta 无法在后台持续运行

### DISCUSSION THREAD

**Discussion by Alanqxy** [MetaCubeX/ClashMetaForAndroid#410] [General]
三星S24 Ultra，系统更新 One UI 6.1 后，Clash Meta 无法在后台持续运行，试过各种版本都不行。

电池选项选择“不受限制”也没用，猜测可能是三星 Knox 安全策略更新的缘故。

安全文件夹内（沙盒）内的 Clash Meta 不受限制

**Comment by blur000x** [comment]
卸载重新安装          后台设置允许后台       下载最新版本的clash      先试试看看        三星21系列         6.1系统可以用     没啥问题

**Reply by Alanqxy** [reply]
都试过了，并没有什么改善
